### PR TITLE
Update to svd2rust 0.21 and form 0.8.

### PR DESCRIPTION
**Pull request by thejpster**

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@ set -ex
 
 # NOTE: Last executed using Rust 1.49.0
```
Comment (henkkuli) [inline update.sh:5]: Just want to make sure this is up to date. And regardless if it is or is not, is this comment even needed?

Comment (9names) [reply update.sh:5]: I think it was intended as a low-effort MSRV, but svd2rust already specifies an MSRV.
We do not lose anything by dropping the comment, so I think we should do that.

**Review by 9names — APPROVED** [review]

File: update.sh (hunk 1 above)
Comment (thejpster) [reply update.sh:5]: The comment is wrong now so should be removed or fixed (I used current stable). I don't think it's useful so I suggest we remove.

Comment (9names) [reply update.sh:5]: I will make a separate PR to affect this change

**Pull request merged**